components/bl-pdfViewer-component/src/index.js

```diff
@@ -123,9 +130,13 @@ export default function PdfViewer({ component, eventHandlers, elRef }) {
 }
 
 const getBottomOffset = el => {
-  const rect = el.getBoundingClientRect();
+  if (el) {
+    const rect = el.getBoundingClientRect();
+
+    return rect.bottom + window.scrollY;
+  }
 
-  return rect.bottom + window.scrollY;
+  return null;
```

Comment:
why null?

For example here:

`const spaceForControls = getBottomOffset(controlsRef.current) - getBottomOffset(pageRef);`

could be `null - 10` - looks strage